docs/Rapid/4-Keyper Manual/4-Adaptive Designer/8-Element picker features/8-Element picker features.md

```diff
@@ -0,0 +1,150 @@
+
+When the **Controls Tab** is active, then you will see six categories of controls:
+
+   a. **All** - This category displays the list of all the controls and templates. Please note, it will also include controls from Connections as well.
+
+   b. **Input** - This category includes all the controls that accept input from a user. For example - Single line, Number etc.  There are 19 controls in this category. 
+   
+   ![Image showing controls in the Input category](<Element Picker 3.png>)  
+
+   :::note[Please note]
+   "User" here is referred to the user of the Adaptive Document, the one for whom the Adaptive Document is intended. For example - the person filling up the lead form on the organisation is the User of the Adaptive Document.  
+   :::
+
+   c. **Control** - This category has two controls. They allow users to interact with the document in a defined manner. Buttons and Progress bar are the two controls in this category.   
+   
+   ![Image showing controls in the Control category](<Element Picker 4.png>)  
+
+   :::warning[Please note]
+   Do not confuse the title of this category with the control object, which can be configured as element. The title of this category is "Control" which derives its name from the interactive actions defined for the user.
+   :::  
+
+   d. **Layout** - This category includes controls that allow you to arrange and organise elements on the document. For example - Section, Accordion etc.  There are 17 controls in this category.
+
+   ![Image showing controls in the Layout category](<Element Picker 5.png>)
+
+   e. **Content** - This category includes any control that displays information on the Adaptive Document. For example - Static Text, Image etc. This category has six controls.
+
+   ![Image showing controls in the Content category](<Element Picker 6.png>)
+
+   f. **Simpli** - This is a special category and does not contain any controls, as such. However, it contains templates of layouts (controls arranged in predefined layouts) available for quick design. For example - Single Page Layout, Accordion Layout. You can select the template that most closes matches with your needs and then add or delete controls on the <a href="https://rapiddocs.z8.web.core.windows.net/docs/Rapid/User%20Manual/glossary/#composition-area" target="_blank">**composition area**</a>. This category has five layout templates.
+
+   ![Image showing templates in the Simpli category](<Element Picker 8.png>)
+
+### Categories in Connections Tab
+
+When **Connections Tab** is active, the list of categories displayed are changed.
+
+![Image showing categories in Connections tab](<Element Picker 9.png>)
+
+a. **All** - This category does not change between Controls Tab and Connections Tab. It displays the list of all the controls and templates available across both the tabs.
+
+b. ***List title*** - You can then see all the list titles for which the connection has been successfully established with the document.
```

Comment:
If the user has renamed their connection, it will display that name instead of the list name.